config/.rubocop.yml

```diff
@@ -1,8 +1,11 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
It has to be in the very top level folder. Cannot be in `config`, needs to be 1 level higher.